transform/transformers/common_software/pck_transformer.py

```diff
@@ -311,7 +311,7 @@ def round_numeric_values(self):
 
         if self.survey.get('survey_id') in [self.qsi_survey_id]:
             decimal.getcontext().rounding = ROUND_HALF_UP
-            self.data.update({k: str(Decimal(round(Decimal(float(v))) / 1000).quantize(1) * 1000)
+            self.data.update({k: str(self.round_to_nearest_thousand(v))
```

Comment:
As `ROUND_HALF_UP` is the default operation for `round_to_nearest_thousand()` you don't need to specify it. I think that makes this call identical to the one below it so could just be combined?
```
if self.survey.get('survey_id') in [self.qcas_survey_id, self.qsi_survey_id]:
    self.data.update({k: str(self.round_to_nearest_thousand(
        for k, v in self.data.items() if k in self.qcas_currency_questions})
```

Reply:
Good spot

Reply:
My bad, misread the end of the function so they're not exactly the same. Too many acronyms lead to confuzzle :)